Server content/Resources/Prototypes/Roles/Jobs/Engineering/builder.yml

```
@@ -0,0 +1,35 @@
+- type: job
+  id: Builder
+  name: job-name-builder
+  description: job-description-builder
+  playTimeTracker: JobBuilder
+  requirements:
+    - !type:DepartmentTimeRequirement
+      department: Engineering
+      time: 21000 # Corvax-RoleTime
+  startingGear: BuilderGear
+  icon: "StationBuilder"
```

```suggestion
  icon: "Builder"
```
Нет такого: StationBuilder